hosting/how-to-host-on-sia.md

```diff
@@ -1,151 +1,239 @@
 
-### Enable Disk Compression \(Windows only, Optional\)
-Windows does not support sparse files by default, which causes downtime when adding large storage folders. This downtime can be avoided by enabling disk compression on any disk that you use for Sia storage. 
+## Preparing Your Storage Drive\(s\) \(Windows Only, Optional\)
 
-![](../.gitbook/assets/host-disk-compression.png)
+NTFS file system of Windows does not support sparse files, which causes downtime when adding large \(several TB\) storage folders. To avoid this downtime, you can format your storage drive using ReFS before adding it to Sia network.
 
-## The process
+Unfortunately, ReFS is not available on all Windows releases. It is currently available on Windows 10 Server, Windows 10 Enterprise, and Windows 10 Pro for Workstations. Of these, Windows 10 Pro for Workstations is the most available solution: it’s very easy to upgrade from Windows 10 Pro, and the upgrade license doesn’t cost much. After all, you will probably earn more Siacoins within the time, which your host would otherwise be spending on filling your storage space with zeroes, like it happens on NTFS.
 
-Open Sia and click on the **Host** tab on the left. This is where you can configure your host settings. The first thing you do is tell Sia where your storage folder is. This is a location on the storage device you want to use for hosting.
+Here is how you can make advantage of ReFS:
 
-![](../.gitbook/assets/hosting-1.png)
+1.	Right-click on your Windows icon and select Disk Management.
+2.	Create an NTFS partition on your new drive \(if this is a new drive\).
+3.	Convert your NTFS partition into ReFS.
 
-Click **Add a folder**, then select a location. You can choose a drive \(like D:\\) to use an entire empty drive or partition, or you can pick a specific folder on a drive. Sia will create one large file in the location you select that allocates the amount of storage you choose.
+![](../.gitbook/assets/host-refs.jpg)
 
-![](../.gitbook/assets/hosting-2.png)
+## Disabling Automatic Updates \(Windows Only, Optional\)
 
-Tell Sia how much space you want to allocate to hosting, in megabytes. Entering a value lower than 32 GB will provide a warning since as a host you're graded on having a lot of available storage. Set at least 4 TB if you can. You can type in a value if you want to set a value higher than what the slider allows.
+Windows updates often bring important security improvements, but if they happen when Windows wants it, and not you, it becomes a nightmare. Your PC can reboot without you knowing it, or the new software may decide that Sia is a virus and remove it. So, it is better to disable automatic updates.
+
+However, it’s not as easy as it sounds. You can stop Windows Update service, but it will start again after a reboot. You can even disable it, but it will become magically enabled again. The solution is to edit the Group Policy.
```

Comment:
> However, it’s not as easy as it sounds. You can stop **the** Windows Update service, but it will start again after a reboot. ~~You can even disable it, but it will become magically enabled again.~~ The solution is to edit the Group Policy.